cachet_url_monitor/configuration.py

```diff
@@ -239,6 +239,25 @@ def push_metrics(self):
             else:
                 self.logger.warning(f'Metric upload failed with status [{metrics_request.status_code}]')
 
+    def trigger_webhooks(self):
+        """Trigger webhooks."""
+        if self.status == st.ComponentStatus.PERFORMANCE_ISSUES:
+            message = self.message
+            title = f'{self.endpoint["name"]} degraded'
+        elif self.status == st.ComponentStatus.OPERATIONAL:
+            message = 'Incident resolved'
+            title = f'{self.endpoint["name"]} OK'
+        else:
+            message = self.message
+            title = f'{self.endpoint["name"]} unavailable'
+        print('hooks', self.webhooks)
```

Comment:
We should use logger here:
```
self.logger.info(f'hooks {self.webhooks}')
```

Reply:
Better remove this debugging code, this is not really useful thing to log.